app/Models/EntryDAO.php

```diff
@@ -873,12 +872,28 @@ public function countNotRead($minPriority = null) {
 	}
 
 	public function countUnreadReadFavorites() {
-		$sql = 'SELECT c FROM ('
-			.	'SELECT COUNT(id) AS c, 1 as o FROM `' . $this->prefix . 'entry` WHERE is_favorite=1 '
-			.	'UNION SELECT COUNT(id) AS c, 2 AS o FROM `' . $this->prefix . 'entry` WHERE is_favorite=1 AND is_read=0'
-			.	') u ORDER BY o';
+		$sql = <<<SQL
+  SELECT c
+    FROM (
+         SELECT COUNT(e1.id) AS c
+              , 1 AS o
+           FROM `{$this->prefix}entry` AS e1
+           JOIN `{$this->prefix}feed` AS f1 ON e1.id_feed = f1.id
+          WHERE e1.is_favorite = 1
+            AND f1.priority >= :priority_normal
+         UNION
+         SELECT COUNT(e2.id) AS c
+              , 2 AS o
+           FROM `{$this->prefix}entry` AS e2
+           JOIN `{$this->prefix}feed` AS f2 ON e2.id_feed = f2.id
+          WHERE e2.is_favorite = 1
+            AND e2.is_read = 0
+            AND f2.priority >= :priority_normal
```

Comment:
I am a bit unsure of what we should do regarding favourites in muted/archived feeds. What do you think?
The same goes for export.

Reply:
I think that if you don't want to see a feed, you probably don't want to see the favorites either. That's why I've added that filter here.